### ISSUE TITLE
WGET command gives 404 error

### ISSUE THREAD

**Issue by donburch888** [rmcrackan/Libation#975]
**Describe the bug**
In your documentation page https://github.com/rmcrackan/Libation/blob/master/Documentation/InstallOnLinux.md the URLs given are incomplete and result in 404 Not Found error. 

It took me a little while to track down the correct info. 

Curiously I note that your latest official release is quite a long way behind your development version.

**Expected behavior**
The filenames should include the hardware platform. 
I suggest a bit more explanation of how to find out which version, and available hardware platforms ... such as adding a link to the latest releases:

> **Install and Run Libation on Ubuntu**
>
>New Libation releases are automatically packed into .deb and .rpm package for different hardware platforms, and are available from the [Libation repository's releases](https://github.com/rmcrackan/Libation/releases/latest) page.
>
>Run this command in your terminal to download and install Libation, replacing the X.X.X with the latest Libation package version number, and PLATFORM with the appropriate values:
>
>- Debian
>
>```
>    wget -O libation.deb https://github.com/rmcrackan/Libation/releases/download/vX.X.X/Libation.X.X.X-linux-chardonnay-PLATFORM.deb &&
>    sudo apt install ./libation.deb
>```

**Platform**

Ubuntu Linux 22.04.4

**Log Files**
```
don@ponder:~$ wget -O libation.deb https://github.com/rmcrackan/Libation/releases/download/v11.1.0/Libation.11.1.0-linux-chardonnay.deb && sudo apt install ./libation.deb
--2024-09-14 11:03:44--  https://github.com/rmcrackan/Libation/releases/download/v11.1.0/Libation.11.1.0-linux-chardonnay.deb
Resolving github.com (github.com)... 4.237.22.38
Connecting to github.com (github.com)|4.237.22.38|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2024-09-14 11:03:44 ERROR 404: Not Found.
```

**Comment by rmcrackan** [comment]
> Curiously I note that your latest official release is quite a long way behind your development version.

Yeah, it's weird I know. v11.1 works for most people. If it doesn't, try the latest pre-release.

**Comment by rmcrackan** [comment]
This bug seems to be out of date. If you have more information, please reply here and we can look into it further.

Please try the latest version: https://github.com/rmcrackan/Libation/releases/latest
